src/main/java/org/testeditor/fixture/host/HostDriverFixture.java

```diff
@@ -284,24 +316,30 @@ private Result setCursorPosition(int row, int col) {
      *            representation.
      * @param locatorType
      *            see {@link LocatorStrategy}
-     * @return {@link Result} of
+     * @return {@link Result} of the action.
      */
     @FixtureMethod
     public Result moveCursor(String elementLocator, LocatorStrategy locatorType) {
         Result result = null;
         Status status = getStatus();
         switch (locatorType) {
         case START:
-            LocatorByStart locatorByStart = new LocatorByStart(elementLocator, status);
-            result = setCursorPosition(locatorByStart.getStartRow(), locatorByStart.getStartColumn());
+            LocatorByStart locatorByStart = new LocatorByStart(elementLocator, status, this.offsetRow,
+                    this.offsetColumn);
+            result = setCursorPosition(locatorByStart.getStartRowWithOffset(),
+                    locatorByStart.getStartColumnWithOffset());
             break;
         case START_STOP:
-            LocatorByStartStop locatorByStartStop = new LocatorByStartStop(elementLocator, status);
-            result = setCursorPosition(locatorByStartStop.getStartRow(), locatorByStartStop.getStartColumn());
+            LocatorByStartStop locatorByStartStop = new LocatorByStartStop(elementLocator, status, this.offsetRow,
+                    this.offsetColumn);
+            result = setCursorPosition(locatorByStartStop.getStartRowWithOffset(),
```

Comment:
That's a lot of repeated code. The creation of the locator should be extracted into a helper method.